common/clock/time_source.go

```diff
@@ -45,7 +46,7 @@ type (
 
 	// EventTimeSource serves fake controlled time
 	EventTimeSource struct {
-		now time.Time
+		now int64
```

Comment:
is EventTimeSource used in multi thread env?

Reply:
Yes, this is why I have to make this conversion. Race detector complained about it.